pyrex.py

```diff
@@ -259,7 +259,13 @@ def build_image(config, build_config):
             build_config["build"]["buildid"] = get_image_id(config, tag)
         except subprocess.CalledProcessError:
             try:
-                engine_args = [engine, "pull", tag]
+                if config["config"]["buildlocal"] == "0":
```

Comment:
This is already in the else case of `config["config"]["buildlocal"] == "1"` (lines 211 and 255), so I think this is redundant